ts/output/svg.ts

```diff
@@ -262,9 +262,9 @@ CommonOutputJax<N, T, D, SVGWrapper<N, T, D>, SVGWrapperFactory<N, T, D>, SVGFon
       adaptor.setStyle(svg, 'min-width', this.ex(W));
       adaptor.setAttribute(svg, 'width', pwidth);
       adaptor.removeAttribute(svg, 'viewBox');
-      const scale = wrapper.metrics.ex / (this.font.params.x_height * 1000);
-      adaptor.setAttribute(g, 'transform', 'matrix(1 0 0 -1 0 0) scale(' +
-                           this.fixed(scale, 6) + ') translate(0, ' + this.fixed(-h * 1000, 1) + ')');
+      const scale = this.fixed(wrapper.metrics.ex / (this.font.params.x_height * 1000), 6);
+      adaptor.setAttribute(g, 'transform', 'scale(' + scale + ',-' + scale + ') '
+                            + 'translate(0, ' + this.fixed(-h * 1000, 1) + ')');
```

Comment:
Can you rewrite that into a string literal.

It appears that string literals are now faster than concatenation on all browsers, so we should try to prefer that.

Reply:
OK, will do.  We will probably need to do a pass over the whole codebase to update that.